### PR TITLE
Add README

### LINKED ISSUE #78: Add README

### PR DESCRIPTION
**Pull request by jiguantong**
- [x] Build from source
- [x] Badges
- [x] Local test net

### CODE REVIEW

**Comment by acgxv** [comment]
How's the progress?

**Comment by boundless-forest** [comment]
Can I pick up this task?

**Comment by jiguantong** [comment]
sure

File: README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+git checkout <latest tagged release>
+cargo build --release
+```
+
+# Connect to existed networks
+
+- Connect to the global Darwinia Parachain Mainnet network
+
+    ```shell
+    ./target/release/darwinia-collator
```
Comment (acgxv) [inline README.md]: The name is `darwinia-parachain`.

Comment (boundless-forest) [reply README.md]: The output of `cargo build --release`:

```sh
$ ls target/release/
build/  darwinia-collator*  darwinia-collator.d  deps/  examples/  incremental/  wbuild/
```

And 

```rs
[[bin]]
name = "darwinia-collator"
path = "src/main.rs"
```

Or I am missing something?

Comment (acgxv) [reply README.md]: Rebase `main`.

Comment (jiguantong) [reply README.md]: https://github.com/paritytech/cumulus/pull/1241

Comment (boundless-forest) [reply README.md]: Thanks for your guys' explanation.  I got it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Darwinia Parachain
+
+[![License](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+[![Substrate version](https://img.shields.io/badge/Substrate-3.0.0-brightgreen?logo=Parity%20Substrate)](https://substrate.io)
+[![Checks](https://github.com/darwinia-network/darwinia-parachain/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/darwinia-network/darwinia-parachain/actions/workflows/ci.yml)
```
Comment (boundless-forest) [inline README.md]: This one is broken, Could somebody help me about it?

**Review by acgxv — APPROVED** [review]

**Pull request merged**